src/ocm/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -46,6 +50,7 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
   const clusterVarieties = useClusterStatusVarieties(cluster);
   const { credentials, credentialsError } = clusterVarieties;
   const { monitoredOperators = [] } = cluster;
+  const canAddHosts = isSingleClusterMode() && !isSNO(cluster) && cluster.status === 'installed';
```

Comment:
@jgyselov why this should apply only if the package was built using single cluster mode? (that mode is quite deprecated)

Reply:
Did I completely misunderstand what this feature was for? I.e. the on-prem env with only one cluster to which we want to be able to add hosts after installation?